templates/2_Pixie_Cluster_Pixels.ipynb

```diff
@@ -31,18 +31,46 @@
     "from ark.utils import data_utils, example_dataset, plot_utils\n",
```

Comment:
<p>Is there a reason you chose to default this as False? I think it'd be good to have the default to be True so new users can run the notebook completely through without changing anything.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>